src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/installer.targets

```
@@ -279,7 +279,7 @@
   -->
   <Target Name="GetCommonJsonProperties">
     <PropertyGroup>
-      <FullLicenseText>$([System.IO.File]::ReadAllText('$(ProjectDir)LICENSE.TXT').Replace('%0A', '\n').Replace('"', '\"'))</FullLicenseText>
+      <FullLicenseText>$([System.IO.File]::ReadAllText('$(LicenseFileLocation)').Replace('%0A', '\n').Replace('"', '\"'))</FullLicenseText>
```

Applying `File` change:

```suggestion
      <FullLicenseText>$([System.IO.File]::ReadAllText('$(LicenseFile)').Replace('%0A', '\n').Replace('"', '\"'))</FullLicenseText>
```